README.md

```diff
@@ -1,47 +1,119 @@
-# Cell morphology analysis
+# Phenotypic Analysis of Interfertile Algal Species
 
 This repository contains code for image processing and analysis to compare phenotypes of interfertile *Chlamydomonas* species. These analyses are included in the publication [Phenotypic differences between interfertile Chlamydomonas species](https://doi.org/10.57844/arcadia-35f0-3e16).
 
 ![Chlamydomonas smithii is larger than C. reinhardtii but they have a similar shape](https://github.com/Arcadia-Science/chlamy-comparison/assets/110641190/a6854e63-e4f6-4779-904e-02ca65fb4d4d)
 
-Release v1.0.1
+Release v2.1
+
+Github
 
 [![DOI](https://zenodo.org/badge/644048016.svg)](https://zenodo.org/badge/latestdoi/644048016)
 
+Data Repository
 
-## Versions and platforms
-*Fiji macro* was used with ImageJ 1.53t
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.8326749.svg)](https://doi.org/10.5281/zenodo.8326749)
 
-*Cellprofiler pipeline* was run on CellProfiler version 4.2.5
+# Computational Protocols
```

Comment:
Before this, I'd include our standard conda installation language and instructions to make sure the users are aware your environment exists

Reply:
Example repo: https://github.com/Arcadia-Science/2023-amblyomma-americanum-txome-assembly